addons/account/models/account_report.py

```diff
@@ -558,6 +558,14 @@ class AccountReportExpression(models.Model):
         ),
     ]
 
+    @api.constrains('carryover_target')
+    def _check_carryover_target(self):
```

Comment:
While we're at it, let's remove the part about 'custom' here https://github.com/odoo/odoo/blob/17.0/addons/account/models/account_report.py#L544 

I checked, and clearly, this is a leftover ; it's not used anywhere, and not even supported in the code anymore. 

(this will need an entry in the pot as well)

Reply:
Okey I was unsure about what it was :)

Reply:
@oco-odoo Is this handled by Transifex or do I need to translate everything ?

Reply:
Just updating the .pot is enough ; Transifex handles the rest, since we're in the account module